### DISCUSSION TITLE
Glossary & DeepL glossary

### DISCUSSION THREAD

**Discussion by cabsil** [WeblateOrg/weblate#11746] [Q&A] [answered]
Hello, 

I enabled the automatic suggestion with DeepL. I saw Weblate is handling a kind of synch between the Weblate glossary and the DeepL glossary. I am able to get sometimes entries synched from Weblate to DeepL but not all the time. I don't understand what triggers the update of the glossary in DeepL. 
For example I was able to get some entries in the glossary but I added more entries but they didn't appear in DeepL glossary.
Could anyone explain how is it working ?

Regards

Christophe Absil

**Comment by nijel** [comment, marked as answer]
The glossary is checked whether it is up to date when automatic translation is used

**Reply by cabsil** [reply]
Hello, 

Thx for the reply. I just tested it and I just went on a string to translate, clicked on suggestions and the glossary was updated in DeepL. To be more precise, the glossary in DeepL is deleted and a new one is created.

**Reply by nijel** [reply]
Yes, there is no API in DeepL to update existing glossary, as those are immutable. Deleting and recreating is the recommended approach to updating one, see https://developers.deepl.com/docs/api-reference/glossaries.
